src/Evolution/Systems/GrMhd/ValenciaDivClean/FixConservatives.cpp

```diff
@@ -327,6 +327,15 @@ bool FixConservatives::operator()(
       }
     }
 
+    else {
+      auto zeroes = static_cast<SimdType>(0);
+      if (const auto tilde_tau_mask = tau_tilde > zeroes;
+          simd::any(tilde_tau_mask)) {
```

Comment:
This looks backwards to me.  Can you add a test?

Reply:
fixed and added the test